requirements.devel.txt

```diff
@@ -52,6 +52,7 @@
 -e git+https://github.com/inveniosoftware/invenio-mail.git#egg=invenio-mail
 # FIXME #75
 # -e git+https://github.com/inveniosoftware/invenio-migrator.git#egg=invenio-migrator
+-e git+https://github.com/inveniosoftware/invenio-oaiserver.git#egg=invenio-oaiserver
```

Comment:
So, just from curiosity why have we deleted this in the first place?

Reply:
it was typo 😝